api/Models/Role.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace api.Models
+{
+    public class Role
+    {
+        [Key]
+        public string RoleId { get; set; }
+        
+        public string RoleName { get; set; }
+        
+        public string NormalizedName { get; set; }
+
+        public ICollection<UserRole> Roles { get; set; }
+    }
+}
```

Comment:
Need to deal with IdentityRole. 
[Reference](https://stackoverflow.com/questions/22057403/extending-identityrole-and-identityuser)